### DISCUSSION TITLE
Custom Questionnaires issue

### DISCUSSION THREAD

**Discussion by Perignea** [globaleaks/globaleaks-whistleblowing-software#3677] [Q&A]
Hi all, has anyone had an issue with questionnaires, please? 
I created a new questionnaire and also did it as the default questionnaire in the settings, but when someone goes to file a report it is not the questionnaire that I created and assigned that is coming up on the screen. Quite new to this so I wanted to see if I am missing a step? 
Thanks

**Comment by mbolzan** [comment]
you need to assign your new questionnaire to your channel, maybe default channel

let see https://github.com/orgs/globaleaks/discussions/3540
